omop/v4/schema/observation_period.csv

```diff
@@ -0,0 +1,5 @@
+model,version,table,field,type
```

Comment:
I notice you're only including `length`, `scale`, and `precision` columns in files that contain that information. I would have thought your fondness for declared structures would lean towards always including the same fields, but I don't mind either way.